server/src/main/java/com/objectcomputing/checkins/services/kudos/KudosServicesImpl.java

```diff
@@ -173,10 +170,6 @@ public KudosResponseDTO getById(UUID id) {
 
     @Override
     public void delete(UUID id) {
-        if (!currentUserServices.isAdmin()) {
-            throw new PermissionException(NOT_AUTHORIZED_MSG);
-        }
-
         Kudos kudos = kudosRepository.findById(id).orElseThrow(() ->
```

Comment:
Same above here. Is there a permission check happening?

Reply:
Same with the delete:

```
    @Delete("/{id}")
    @Status(HttpStatus.NO_CONTENT)
    @RequiredPermission(Permission.CAN_ADMINISTER_KUDOS)
    public void delete(@NotNull UUID id) {
        kudosServices.delete(id);
    }
```